sdk/deviceupdate/arm-deviceupdate/src/models/index.ts

```diff
@@ -524,7 +524,7 @@ export interface Account extends TrackedResource {
   /** List of private endpoint connections associated with the account. */
   privateEndpointConnections?: PrivateEndpointConnection[];
   /** Device Update Sku */
-  sku?: Sku;
+  sku?: SKU;
```

Comment:
This breaking is introduced by the fix : https://github.com/Azure/autorest.typescript/issues/1404.